### PR TITLE
[unit-tests] unit tests for SizeInfo component

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Adds unit  tests for `SizeInfo` component

## Related Issue
- Part of #5234

## Motivation and Context
- increase unit tests coverage

## How Has This Been Tested?
- test environment: local

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/src/components/AppBar/SelectedResources/SizeInfo.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="uk-flex uk-flex-middle">
     <translate
       v-if="selectedResourcesSize !== '?'"
-      key="multiple-select-info"
+      key="multiple-select-info-with-size"
```
Comment (saw-jan) [inline packages/web-app-files/src/components/AppBar/SelectedResources/SizeInfo.vue:5]: this section shows number of selected along with size so I think the key should be `multiple-select-info-with-size`

File: packages/web-app-files/src/components/AppBar/SelectedResources/SizeInfo.vue [hunk 2]
```diff
@@ -11,7 +11,7 @@
     </translate>
     <translate
       v-else
-      key="multiple-select-info-with-size"
+      key="multiple-select-info"
```
Comment (saw-jan) [inline packages/web-app-files/src/components/AppBar/SelectedResources/SizeInfo.vue:14]: this section only shows number of selected so I think the key should be `multiple-select-info`

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/17377/27/1
The following scenarios passed on retry:
- webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:154

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/17390/27/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternalRoot</strong> https://drone.owncloud.com/owncloud/web/17395/39/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by kiranparajuli589 — COMMENTED** [review]
Lgtm

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/17396/11/1
The following scenarios passed on retry:
- webUIFavorites/unfavoriteFile.feature:34

File: packages/web-app-files/tests/unit/components/AppBar/SelectedResources/SizeInfo.spec.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+const componentStubs = { ...stubs, translate: true }
+
+describe('SizeInfo component', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  jest
+    .spyOn(SizeInfo.mixins[0].methods, 'getResourceSize')
+    .mockImplementation(size => (isNaN(size) ? '?' : size + ' B'))
```
Comment (individual-it) [inline packages/web-app-files/tests/unit/components/AppBar/SelectedResources/SizeInfo.spec.js:39]: I don't feel good about having extra logic in the mock. Can we have different mocks for different cases? That would make the tests easier to read and have less risk that the logic in the tests break

**Review by individual-it — CHANGES_REQUESTED** [review]
@LukasHirt @kulmann can you have a look if the code-changes are OK

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/17480/27/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Review by individual-it — COMMENTED** [review]
tests look good!

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsers</strong> https://drone.owncloud.com/owncloud/web/17487/24/1
The following scenarios passed on retry:
- webUISharingInternalUsersShareWithPage/shareWithUsers.feature:91

**Review by LukasHirt — APPROVED** [review]
Code changes LGTM

**Review by individual-it — APPROVED** [review]

**Pull request merged**